.circleci/config.yml

```
@@ -217,121 +173,133 @@ commands:
             python -V
             pip -V
             kedro -V
+      - run:
+          name: Install 'make' command
+          command: choco install make
+
+
+  run_win_python_tests:
+    steps:
+      - run:
+          name: Run Python tests on Windows
+          command:
+            conda activate kedro-viz; make pytest
+
+  run_win_e2e_tests:
+    steps:
+      - run:
+          name: Run all end to end tests on Windows
+          command:
+            conda activate kedro-viz; make e2e-tests
 
-  win_build:
-    description: Run build on Windows
+
+jobs:
+  e2e_tests:
     parameters:
       python_version:
         type: string
+    executor:
+      name: docker
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup_python_env
+      - install_node_dependencies
+      - run_e2e_tests
```

After removing the `run_e2e_tests` commands:
```suggestion
      - run:
          name: Run Python tests
          command: make pytest
```